src/test/java/io/dockstore/provision/DOSPluginUnitTest.java

```diff
@@ -56,30 +103,153 @@ public void testPrepareDownload() throws IOException {
 
         InputStream mockInputStream = IOUtils.toInputStream(
         "{" +
-                "\"data_object\": {" +
-                        "\"checksums\": [" +
-                                "{" +
-                                    "\"checksum\": \"bf140ba9778dd091d533c59d888c4463\"," +
-                                    "\"type\": \"md5\"" +
-                                "}" +
-                        "]," +
-                        "\"created\": \"2018-05-26T13:43:17.056861\"," +
-                        "\"description\": \"\"," +
-                        "\"id\": \"dg.4503/1aad0eb6-0d89-4fdd-976c-f9aa248fc88c\"," +
-                        "\"mime_type\": \"\"," +
-                        "\"name\": null," +
-                        "\"size\": 1503901," +
-                        "\"updated\": \"2018-05-26T13:43:17.056871\"," +
-                        "\"urls\": [" +
-                                "{" +
-                                        "\"url\": \"gs://cgp-commons-multi-region-public/topmed_open_access/53ae11a8-ef4d-5aa6-9227-f372b422f5a1/NWD446684.recab.cram.crai\"" +
-                                "}," +
-                                "{" +
-                                        "\"url\": \"s3://cgp-commons-public/topmed_open_access/53ae11a8-ef4d-5aa6-9227-f372b422f5a1/NWD446684.recab.cram.crai\"" +
-                                "}" +
-                        "]," +
-                        "\"version\": \"89dfdc16\"" +
-                "}" +
+            "\"data_object\": {" +
+                "\"checksums\": [" +
+                    "{" +
+                        "\"checksum\": \"bf140ba9778dd091d533c59d888c4463\"," +
+                        "\"type\": \"md5\"" +
+                    "}" +
+                "]," +
+                "\"created\": \"2018-05-26T13:43:17.056861\"," +
+                "\"description\": \"\"," +
+                "\"id\": \"dg.4503/1aad0eb6-0d89-4fdd-976c-f9aa248fc88c\"," +
+                "\"mime_type\": \"\"," +
+                "\"name\": null," +
+                "\"size\": 1503901," +
+                "\"updated\": \"2018-05-26T13:43:17.056871\"," +
+                "\"urls\": [" +
+                    "{" +
+                        "\"url\": \"gs://cgp-commons-multi-region-public/topmed_open_access/53ae11a8-ef4d-5aa6-9227-f372b422f5a1/NWD446684.recab.cram.crai\"" +
+                    "}," +
+                    "{" +
+                        "\"url\": \"s3://cgp-commons-public/topmed_open_access/53ae11a8-ef4d-5aa6-9227-f372b422f5a1/NWD446684.recab.cram.crai\"" +
+                    "}" +
+                "]," +
+                "\"version\": \"89dfdc16\"" +
+            "}" +
+        "}");
+
+        BufferedReader in = new BufferedReader(new InputStreamReader(mockInputStream));
```

Comment:
This is a test so it doesn't really matter. 
But you should use https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html or remember to close the BufferedReader